src/ControllerRunner.ts

```diff
@@ -12,7 +12,7 @@ import {ResponseInterceptorInterface} from "./interceptor/ResponseInterceptorInt
 import {ResponseInterceptorMetadata} from "./metadata/ResponseInterceptorMetadata";
 import {ResultHandleOptions} from "./ResultHandleOptions";
 
-export type Container = { get(someClass: any): any };
+export interface Container { get(someClass: any): any };
```

Comment:
got it, thanks. I used type because I expected that users will use only structural matching of the types. But having interface maybe use useful for some users too